README.md

```diff
@@ -1,381 +1,246 @@
+[![github-code](https://img.shields.io/badge/GitHub-code-informational?logo=github)](https://github.com/neptune-ai/examples/tree/main/integrations-and-supported-tools/pytorch-lightning/scripts)
+[![jupyter-code](https://img.shields.io/badge/Jupyter-code-informational?logo=jupyter)](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb)
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neptune-ai/examples/blob/main/integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb)
+[<img src="https://img.shields.io/badge/docs-PyTorch%20Lightning-yellow">](https://docs.neptune.ai/integrations/lightning/)
 &nbsp;
 
-[![github-code](https://img.shields.io/badge/GitHub-code-informational?logo=github)](https://github.com/neptune-ai/examples/tree/main/integrations-and-supported-tools/optuna/scripts)
-[![jupyter-code](https://img.shields.io/badge/Jupyter-code-informational?logo=jupyter)](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/optuna/notebooks/Neptune_Optuna_integration.ipynb)
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/optuna/notebooks/Neptune_Optuna_integration.ipynb)
-[<img src="https://img.shields.io/badge/docs-Optuna-yellow">](https://docs.neptune.ai/integrations/optuna/)
-
-### Neptune.ai is trusted by great companies
+&nbsp;
+## neptune.ai is trusted by great companies
 &nbsp;
 
 <div align="center">
-    <img src="https://neptune.ai/wp-content/uploads/Neptune-customers.png" width="900" />
+    <img src="https://neptune.ai/wp-content/uploads/2022/11/github-customers.png" width="900" />
 </div>
 &nbsp;
 
 Read how various customers use Neptune to [improve their workflow](https://neptune.ai/customers).
+&nbsp;
```

Comment:
Why do we need to add HTML-like whitespace? Does Markdown doesn't allow multiple newlines?

Reply:
Should I use <br> </br>?